### PR TITLE
MGDOBR-765 - Fix catalog

### PR DESCRIPTION
**Pull request by r00ta**
[MGDOBR-765](https://issues.redhat.com/browse/MGDOBR-765 )

I did not know it was extremely painful to list a resource directory inside a jar. The previous implementation was running fine with mvn and in the tests but not with a jar. 

## PR owner checklist

Please make sure that your PR meets the following requirements:

- [ ] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config).  
  *Run `mvn clean verify -DskipTests` so that imports are correctly set.*
- [ ] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [ ] The layers in the `kustomize` folder have been updated accordingly.
- [ ] All new functionality is tested
- [ ] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

## PR reviewer(s) checklist

- [ ] Check if code and Github action workflows have been modified and if the modification is safe
- [ ] If the modification is safe, add the `safe to test` label

<details>
<summary>
How to trigger pipelines and use the bots:
</summary>

- <b>Run the end to end pipeline</b>  
  Annotate the pull request with the label: `safe to test`. If you want to run the pipeline again, remove and add it again.

- <b>Rebase the pull request</b>  
  Comment with: `/rebase`.
- <b>Deploy BOT</b>
  Comment with `/deploy <dev,stable> when the PR has been merged to deploy to a target environment.

</details>

### CODE REVIEW

File: manager/src/main/resources/schemas/actions/catalog.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[
+  "KafkaTopic.json",
+  "SendToBridge.json",
+  "Slack.json",
+  "Webhook.json"
```
Comment (danielezonca) [inline manager/src/main/resources/schemas/actions/catalog.json:5]: Can you add a test case to verify if this file matches the list of the resources?

File: manager/src/main/resources/schemas/sources/catalog.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[
+  "AwsS3.json",
+  "AwsSqs.json",
+  "Slack.json"
```
Comment (danielezonca) [inline manager/src/main/resources/schemas/sources/catalog.json:4]: Same here

**Review by sutaakar — APPROVED** [review]

File: manager/src/main/resources/schemas/actions/catalog.json (hunk 1 above)
Comment (r00ta) [reply manager/src/main/resources/schemas/actions/catalog.json:5]: done

**Review by danielezonca — APPROVED** [review]

**Review by kostola — APPROVED** [review]
LGTM

**Pull request merged**